apps/web/src/pages/quick-start/components/LoaderProceedTernary.tsx

```diff
@@ -0,0 +1,34 @@
+import { Center, Loader } from '@mantine/core';
+import { Button, colors } from '../../../design-system';
+import { CheckCircle } from '../../../design-system/icons';
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+export function LoaderProceedTernary({
```

Comment:
This component maybe will be redundant if we will do the loader from the new design (loader line on the top of the div)

Reply:
Was decided to leave it as is.